src/ansys/aedt/core/filtersolutions_core/ideal_response.py

```diff
@@ -424,8 +429,10 @@ def s_parameters(
             of the defined frequency ranges, and the second is a
             list of the requested parameters.
         """
-        self.graph_setup.minimum_frequency = minimum_frequency
-        self.graph_setup.maximum_frequency = maximum_frequency
+        if minimum_frequency is not None:
```

Comment:
Same thing here and same question about the unit test.

Reply:
Done